src/components/Icons/facebook.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
i like this approach for icons, gonna update the canarie ticket about SVGs to point to this example.

Reply:
I liked it too, I borrowed it from argo :)